packages/instant/src/components/instant_heading.tsx

```diff
@@ -113,20 +113,23 @@ export class InstantHeading extends React.Component<InstantHeadingProps, {}> {
     }
 
     private readonly _renderEthAmount = (): React.ReactNode => {
+        const ethAmount = format.ethBaseUnitAmount(
+            this.props.totalEthBaseUnitAmount,
+            4,
+            <AmountPlaceholder isPulsating={false} color={PLACEHOLDER_COLOR} />,
+        );
```

Comment:
I'm a little confused about this comment, would you mind clarifying over Slack when you have time?

Reply:
Sorry I could have been clearer. 

In our chat I remember we said something along the lines of "lets make this work for ETH up to $10,000". Meaning that, if ETH is worth $10,000, we don't want to show the "less than" sign until the ETH value is worth less than $0.01. As far as I can tell, this is the `minUnitAmountToDisplay` you chose.

All I am saying now is: can we make `minUnitAmountToDisplay` dynamic since we do actually know the price of ETH?